app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -244,29 +244,35 @@ export default {
   <div>
     <v-row class="mt-6" no-gutters>
       <v-col>
-        <v-row>
-          <v-col cols="12" sm="6" order="2" order-sm="1">
+        <div
+          class="mt-6 d-flex flex-md-row justify-space-between flex-sm-column-reverse flex-xs-column-reverse gapRow"
+        >
+          <div>
             <h1>
               {{ $t('trans.exportSubmissions.exportSubmissionsToFile') }}
             </h1>
-          </v-col>
-          <v-col class="text-right" cols="12" sm="6" order="1" order-sm="2">
+          </div>
+          <div>
             <span>
               <v-tooltip location="bottom">
                 <template #activator="{ props }">
                   <router-link
                     :to="{ name: 'FormSubmissions', query: { f: form.id } }"
                   >
-                    <v-btn class="mx-1" color="primary" icon v-bind="props">
-                      <v-icon icon="mdi:mdi-list-box-outline"></v-icon>
-                    </v-btn>
+                    <v-btn
+                      color="primary"
+                      v-bind="props"
+                      size="x-small"
```

Comment:
The button icon is now very small and hard to see
![image](https://github.com/jasonchung1871/common-hosted-form-service/assets/101672465/db853acf-da92-4caf-84ee-0457b656f8a9)